testing/citest/spinnaker_testing/jenkins_agent.py

```diff
@@ -93,15 +93,15 @@ def export_to_json_snapshot(self, snapshot, entity):
 
 class JenkinsAgent(base_agent.BaseAgent):
   """A specialization of BaseAgent for interacting with Jenkins."""
-  def __init__(self, baseUrl, auth_path, owner_agent, logger=None):
+  def __init__(self, baseUrl, auth_path, owner_agent, logger=None, max_wait_secs=780):
     super(JenkinsAgent, self).__init__(logger=logger)
     self.__http_agent = http_agent.HttpAgent(baseUrl)
     self.__owner_agent = owner_agent
 
     # Allow up to 13 minutes to wait on operations.
```

Comment:
Should the comment be moved to the parameter above?

Reply:
Good point; I tried moving it up and it looked a bit overly-prominent when above the whole function.  I ended up leaving it where it is, but re-worded it in a new commit so it's more clear that the default is set above.